src/main/java/com/parth/jwt/config/WebSecurityConfig.java

```diff
@@ -0,0 +1,42 @@
+package com.parth.jwt.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.config.annotation.authentication.configuration.AuthenticationConfiguration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.http.SessionCreationPolicy;
+import org.springframework.security.web.SecurityFilterChain;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
+
+@Configuration
+@EnableWebSecurity
+public class WebSecurityConfig {
+    @Autowired
+    private JwtAuthenticationEntryPoint jwtAuthenticationEntryPoint;
+
+    @Autowired
+    private JwtAuthenticationFilter jwtAuthenticationFilter;
+
+    @Bean
+    public AuthenticationManager authenticationManager(AuthenticationConfiguration builder) throws Exception {
+        return builder.getAuthenticationManager();
+    }
+
+
+    public SecurityFilterChain securityFilterChain(HttpSecurity httpSecurity)throws Exception{
+        httpSecurity.csrf(csrf -> csrf.disable())
+                .authorizeRequests()
+//                .requestMatchers("/login").permitAll()
```

Comment:
Is the login point open?
How is the user going to access it if it is not open?

Reply:
i was trying out with closing login end point 
it still works fine
now I opened login end point